src/plugins/navigation/review/navigator.js

```diff
@@ -719,4 +743,6 @@ function navigatorFactory(config, map, context) {
     return navigator.init(config);
 }
 
+Handlebars.registerPartial('bubbleButton', bubbleButton);
```

Comment:
**nitpick:** We should avoid adding partials for nested components. Partials are global, and having a component registering specific partial may introduce conflicts and unpredictable issues without having a proper LDS way of doing it. While this may be a good practice in a small project, it is not the case here due to the complexity.

It would be better to either inline the partial (it is used only once for the time being), or to render the element just in time.

Reply:
Unfortunately https://handlebarsjs.com/guide/partials.html#inline-partials implemented in version 4 i just moved this template inside `navigatorBubbles.tpl` and remove imports and partials